seguro/common/broker.py

```diff
@@ -86,17 +86,35 @@ def __init__(
                 exc,
             )
 
+        self.start_listening()
+
+    def __del__(self):
+        self.stop_listening()
+
     def __on_connect(self, client, userdata, flags, rc):
         self.logger.info("Connected with result code %i", rc)
 
     def __on_message(self, client, userdata, msg):
         self.logger.debug("Receive msg: %s - %s", msg.topic, str(msg.payload))
         self.message_queue.put(msg)
 
-    def subscribe(self, topic):
-        """Subscribe client to given topic."""
+    def subscribe(self, topic, callback=""):
```

Comment:
The default argument of `callback `should not be string. I would use `None `here.

Reply:
Good catch, fixed in 0bf70b9feb8e42c6e4bb7b87ef8d98a16c408b9d